### PR TITLE
fix(staging): infinite loading spinner and publish button

### PR DESCRIPTION
**Pull request by gunar**
Fixes two issues [reported on Slack](https://opensystemslab.slack.com/messages/C01E3AC0C03/p1667301408714349?thread_ts=1667300494.749479&cid=C01E3AC0C03). 

Note to self: Re-run the sync script after merging this to fix staging.

### CODE REVIEW

**Comment by gunar** [comment]
[Infinite loading spinner on Staging](https://trello.com/c/ObRfMPjS/2186-infinite-loading-spinner-on-staging)

**Comment by gunar** [comment]
[Can't publish on Staging](https://trello.com/c/oReJXzxD/2185-cant-publish-on-staging)

File: scripts/seed-database/upsert-production-flows.js [hunk 1]
```diff
@@ -72,7 +71,12 @@ const LOCAL_GRAPHQL_ADMIN_SECRET = process.env.HASURA_GRAPHQL_ADMIN_SECRET;
 
     await localClient.request(
       insertFlowsMutation,
-      { flows: productionFlows || [], }
+      {
+        flows: productionFlows?.map(flow => ({
+          ...flow,
+          version: 1,
```
Comment (jessicamcinchak) [inline scripts/seed-database/upsert-production-flows.js:77]: pizza seeds seem to be working fine here, but i'm still a little confused about this line: if you're always setting `version: 1` here, won't that fail to then link up with the most recent operation that you've fetched & inserted? or has that operation's version also been overwritten to 1?

**Review by jessicamcinchak — APPROVED** [review]
thanks for debugging this! left one question, but trust that you've tested and since staging is unusable currently, happy for this merge asap

File: scripts/seed-database/upsert-production-flows.js (hunk 1 above)
Comment (gunar) [reply scripts/seed-database/upsert-production-flows.js:77]: This script isn't doing an UPSERT anymore but instead a DELETE-THEN-INSERT.

**Pull request merged**